.yaspellerrc.json

```diff
@@ -52,7 +52,9 @@
     "Whitepaper",
     "allowlist[s]?",
     "announcementType",
-    "blockchain",
+    "archiveEntryID",
+    "asynchronously",  
+    "(a )?blockchain",
```

Comment:
Non-blocking, but this line should be removed.
```suggestion
```

Reply:
You would think so, right? I actually filed a bug for yaspeller on this: https://github.com/hcodes/yaspeller/issues/187 (unfortunately yaspeller as a project seems to have gone dark about the same time Russia invaded Ukraine).